### ISSUE TITLE
OM Compatibility

### ISSUE THREAD

**Issue by marcelofcastro** [OpenIPSL/OpenIPSL#245]
Models such as:
- [ ] PSS2A 
- [ ] PSS2B 
- [ ] WT4E 
- [x] EXST1 
- [ ] STABNI 
- [ ] ESAC2A
- [ ] GGOV1
- [ ] GGOVDU1

are not working in OpenModelica or struggle numerically during simulation.

Need to fix it for compatibility with open-source tool.

**Comment by marcelofcastro** [comment]
ESAC2A should also be investigated.

**Comment by marcelofcastro** [comment]
GGOV1 and GGOVDU1 should have their implementation enhanced (probably with the better implementation of basic blocks), as discussed in PR #290

**Comment by dietmarw** [comment]
@marcelofcastro Any chance to have another look or decided if this is actually still relevant for the 3.1.0 release (or at all).

**Comment by marcelofcastro** [comment]
Yes! I want to take a look at it again since we had some re-implementation of basic blocks. It might not be relevant to block the release, but it is definitely worth checking it.

**Comment by dietmarw** [comment]
So should this issue be bumped to milestone:3.2.0?
